<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabCopyrightNotice</class>
 <widget class="QWidget" name="TabCopyrightNotice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelCopyrightTitle">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="style" stdset="0">
      <string>normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCopyrightContent1">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="style" stdset="0">
      <string>normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAboutTile">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="style" stdset="0">
      <string>normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAboutContent1">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="style" stdset="0">
      <string>normal</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
